Code/GraphMol/SmilesParse/test.cpp

```diff
@@ -4066,6 +4066,51 @@ void testGithub1925() {
   BOOST_LOG(rdInfoLog) << "done" << std::endl;
 }
 
+
+void testdoRandomSmileGeneration() {
+  BOOST_LOG(rdInfoLog) << "-------------------------------------\n";
+  BOOST_LOG(rdInfoLog)
+      << "Testing the random Generator for SMILES\n"
+      << std::endl;
+  {
+    std::srand(1); // be sure we use it for testcase!
```

Comment:
it's there already using  std::srand(1); it's not the same ?

Reply:
sorry, I missed that